Docs/SupportedAPIs.md

```diff
@@ -69,3 +69,55 @@ Contributions to expand support to unimplemented functionality are always welcom
 | POST   | `/session/:sessionId/window/:windowHandle/maximize` | Supported    | Not implemented     |
 | GET    | `/session/:sessionId/window_handle`                 | Supported    | Not implemented     |
 | GET    | `/session/:sessionId/window_handles`                | Supported    | Not implemented     |
+
+
+## Example Usage
+
+### 1. Installing Swift
+Swift-WebDriver works with Swift 3 and onwards. If you haven't already, install the [most recent release of swift here](https://www.swift.org/install/windows/#installation-via-windows-package-manager). Verify installation with `swift -v` in the terminal
```

Comment:
I don't think that we should claim Swift 3, but rather Swift 5.9.  The Windows port only has been included since 5.3.